### PR TITLE
docs(add-cookooks-docs-equinix): Add Equinix Cookbook

### PR DESCRIPTION
**Pull request by Ivo-Delmis**
SXT-770

Signed-off-by: Ivo Delmis <ivo@blockchaintp.com>

### CODE REVIEW

File: docs/equinix/index.md [hunk 1]
```diff
@@ -1,2 +1,171 @@
...
+* Under user profile set up "Personal SSH Keys" for server access
+* [Optionally set up Personal API Keys](https://metal.equinix.com/developers/api/)
+* Create "New Project" (we used one Project to deploy all servers in all locations)
+
+## Provisioning Equinix Servers
+
+* Select Servers/On Demand
+* Choose a location (Amsterdam, Chicago, Dallas, etc.)
+* Choose a Server Type (we used `c3.small.x86`)
+* Choose an Operating System (we tested RKE2 on `Ubuntu 18.04LTS` and `Ubuntu 20.04LTS`)
```
Comment (duncanjw) [inline docs/equinix/index.md]: Do we have a recommendation?

File: docs/equinix/index.md [hunk 2]
```diff
@@ -1,2 +1,171 @@
...
+* Select number of servers and server names (we used 3-6 servers, with the default hostnames)
```
Comment (duncanjw) [inline docs/equinix/index.md]: Need to be clearer: For our project we used 3 servers to create a Kubernetes cluster to host Sextant and other management tools and minimum of 4 servers in a given region to create a Kubernetes cluster on which to deploy a blockchain network.

File: docs/equinix/index.md [hunk 3]
```diff
@@ -1,2 +1,171 @@
-# Equinix Metal Cookbooks
-### Recipes for using and deploying DLTs using Sextant on Equinix Metal
\ No newline at end of file
+# Equinix Metal Cookbook
```
Comment (duncanjw) [inline docs/equinix/index.md:1]: Actually this is Kubernetes on Equinix Metal Cookbook

File: docs/equinix/index.md [hunk 4]
```diff
@@ -1,2 +1,171 @@
...
+
+Recipe for using and deploying blockchain networks using Sextant on Equinix Metal
```
Comment (duncanjw) [inline docs/equinix/index.md:3]: Actually this is just for setting up Kubernetes on Equinix Metal

Comment (duncanjw) [inline docs/equinix/index.md]: Technically this is laying the groundwork as here you are really providing instructions for (a) creating Kubernetes clusters and (b) installing Sextant (currently missing from this - see comment below)

File: docs/equinix/index.md [hunk 5]
```diff
@@ -1,2 +1,171 @@
...
+```bash
+helm repo add metallb https://metallb.github.io/metallb
+```
+
+* Install MetalLB
+
+```bash
+kubectl create namespace metallb-system
+helm -n metallb-system install metallb metallb/metallb -f <config file from step 2.yaml>
+```
```
Comment (duncanjw) [inline docs/equinix/index.md:342]: Need to add a new section Installing Sextant noting that this requires access to evaluation copy of Sextant for Enterprise. We will create a link for you to include here.

**Review by duncanjw — CHANGES_REQUESTED** [review]
See my comments

File: docs/equinix/index.md (hunk 1 above)
Comment (Ivo-Delmis) [reply docs/equinix/index.md]: Actually, we have no recommendation - it works fine on any platform we tried (including CentOS/RHEL).

File: docs/equinix/index.md (hunk 4 above)
Comment (Ivo-Delmis) [reply docs/equinix/index.md]: Changed.

File: docs/equinix/index.md (hunk 1 above)
Comment (Ivo-Delmis) [reply docs/equinix/index.md]: I've added link to RKE2 docs that has all supported version listed.

File: docs/equinix/index.md (hunk 2 above)
Comment (Ivo-Delmis) [reply docs/equinix/index.md]: Updated.

File: docs/equinix/index.md (hunk 5 above)
Comment (Ivo-Delmis) [reply docs/equinix/index.md:342]: Soudns good.

File: docs/equinix/index.md [hunk 6]
```diff
@@ -1,2 +1,196 @@
...
-### Recipes for using and deploying DLTs using Sextant on Equinix Metal
\ No newline at end of file
+# Equinix Metal Cookbook
+
+Recipe for setting up a Kubernetes cluster on Equinix Metal and installing Sextant
+for using and deploying blockchain networks.
+
+## Setting up for Equinix Metal
+
+* [Log in to Equinix Metal Portal](https://console.equinix.com/)
```
Comment (duncanjw) [inline docs/equinix/index.md]: Log in to [Link]

File: docs/equinix/index.md [hunk 7]
```diff
@@ -1,2 +1,196 @@
...
+* Under user profile, set up "Personal SSH Keys" for server access
+* [Optionally set up Personal API Keys](https://metal.equinix.com/developers/api/)
```
Comment (duncanjw) [inline docs/equinix/index.md]: Optionally set up [Link]

File: docs/equinix/index.md [hunk 8]
```diff
@@ -1,2 +1,196 @@
...
+* Create "New Project" (we used one project to deploy all servers in all
+  locations)
+
+## Provisioning Equinix Servers
```
Comment (duncanjw) [inline docs/equinix/index.md]: ... for Kubernetes

File: docs/equinix/index.md [hunk 9]
```diff
@@ -1,2 +1,196 @@
...
+
+* Select Servers/On-Demand
+* Choose a location (Amsterdam, Chicago, Dallas, etc.)
+* Choose a Server Type (we used `c3.small.x86`)
+* Choose an Operating System. [RKE documentation lists supported
+  versions](https://docs.rke2.io/install/requirements/) (we tested RKE2 on
+  `Ubuntu 18.04LTS` and `Ubuntu 20.04LTS`)
+* Select the number of servers and server names. We recommend using at least three servers nodes for HA, and any number after can be used as agent nodes.
```
Comment (duncanjw) [inline docs/equinix/index.md]: ... three servers as controllers for HA when creating Kubernetes clusters, ...

File: docs/equinix/index.md [hunk 10]
```diff
@@ -1,2 +1,196 @@
...
+  (we used three servers for the Admin cluster for Sextant and between 5-6 for
```
Comment (duncanjw) [inline docs/equinix/index.md]: (In our project ...

File: docs/equinix/index.md [hunk 11]
```diff
@@ -1,2 +1,196 @@
...
+  the three blockchain network clusters).
+* Optionally "Add user data" (handy feature to customize server provisioning)
+* Optionally "Configure IPs" (we kept defaults)
+* Optionally "Customize SSH keys" (we are using keys already configured for the
+  project)
+
+## Set up BGP
+
+* [Set up Local BGP for the project](https://metal.equinix.com/developers/docs/networking/local-global-bgp/)
```
Comment (duncanjw) [inline docs/equinix/index.md]: Set up [Link] for the project

File: docs/equinix/index.md [hunk 12]
```diff
@@ -1,2 +1,196 @@
...
+
+* Install MetalLB
+
+```bash
+kubectl create namespace metallb-system
+helm -n metallb-system install metallb metallb/metallb \
+  -f <config file from step 2.yaml>
+```
+
+## Setting up Sextant
```
Comment (duncanjw) [inline docs/equinix/index.md]: Installing Sextant on Equinix Metal

File: docs/equinix/index.md [hunk 13]
```diff
@@ -1,2 +1,196 @@
...
+
+[Link to Sextant Install goes here](https://docs.blockchaintp.com/en/latest/sextant/daml/overview/)
```
Comment (duncanjw) [inline docs/equinix/index.md]: As discussed let's do the instructions inline here. Start with e.g.

Having created a Kubernetes cluster we can now install Sextant ...

File: docs/equinix/index.md [hunk 14]
```diff
@@ -1,2 +1,196 @@
-# Equinix Metal Cookbooks
-### Recipes for using and deploying DLTs using Sextant on Equinix Metal
\ No newline at end of file
+# Equinix Metal Cookbook
+
+Recipe for setting up a Kubernetes cluster on Equinix Metal and installing Sextant
+for using and deploying blockchain networks.
+
```
Comment (duncanjw) [inline docs/equinix/index.md:5]: Add section on the tools you will need - kubectl and helm ...???

**Review by duncanjw — CHANGES_REQUESTED** [review]
As discussed. Looks like I left this hanging.

**Review by duncanjw — APPROVED** [review]
LGTM

**Pull request merged**